lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_index.html.ep

```diff
@@ -0,0 +1,12 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+% title 'OBS synchronization folders';
+<H2><%= 'OBS synchronization folders ' %></H2>
+% for my $folder (@$folders) {
+	<label><%= link_to "$folder"  => url_for('plugin_obs_rsync_folder', folder => $folder) %></label>
+	<br>
```

Comment:
These `<br>` elements you use *all over the place* start to look wrong. Use block elements where applicable.